src/app/products/components/product-delete-confirm-dialog/product-delete-confirm-dialog.harness.ts

```diff
@@ -0,0 +1,44 @@
+import { ComponentHarness, TestElement } from "@angular/cdk/testing";
+import { MatButtonHarness } from "@angular/material/button/testing";
+
+export class ProductDeleteConfirmDialogHarness extends ComponentHarness {
+    static hostSelector = "tk-product-delete-confirm-dialog";
+
+    getTitle = this.locatorFor("[mat-dialog-title]");
```

Comment:
please use MatDialogHarness instead of this locator. It has methods to retrieve title.
https://material.angular.io/components/dialog/api#MatDialogHarness

Reply:
The methods for getting title and content text were added in 13+, they are not available in 11.2.8 that is in the project.

Reply:
I'll leave it as is for now, unless you require to update everything to ^13.0.0